clients/cobol-lsp-vscode-extension/patches/iconv-lite+0.6.3.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/node_modules/iconv-lite/encodings/sbcs-data.js b/node_modules/iconv-lite/encodings/sbcs-data.js
```

Comment:
I still don't understand of this exact file

Reply:
Well, okay. Anton explained it to me. Is it the only way to do that?

Reply:
No actually.
We can raise a PR on iconv-lite. But I don't think it would be merged. 
Vit showed me this PR from 2015 - https://github.com/ashtuchkin/iconv-lite/pull/112

Reply:
Nah, okay